src/worker/ghidra/ghidra.cc

```diff
@@ -0,0 +1,141 @@
+#include <sleigh/loadimage.hh>
+
+#include <iostream>
+#include <exception>
+#include <typeinfo>
+#include <stdexcept>
+
+#include "sleighMishegos.hh"
+#include "../worker.h"
+#include "mish_common.h"
+
+extern "C" {
+char *worker_name = (char *)"ghidra";
+
+void worker_ctor();
+
+void try_decode(decode_result *result, uint8_t *raw_insn, uint8_t length);
+}
+
+// This is a tiny LoadImage class which feeds the executable bytes to the translator
+class MyLoadImage : public LoadImage {
+  uintb baseaddr;
+  int4 length;
+  uint1 *data;
+
+public:
+  MyLoadImage(uintb ad, uint1 *ptr, int4 sz) : LoadImage("nofile") {
+    baseaddr = ad;
+    data = ptr;
+    length = sz;
+  }
+  virtual void loadFill(uint1 *ptr, int4 size, const Address &addr);
+  virtual string getArchType(void) const {
```

Comment:
I'd prefer using `override` keyword. That way if the function signature deviates from the one in `LoadImage`, we'll get a compiler warning.